### ISSUE TITLE
upgrade error

### ISSUE THREAD

**Issue by tikismoke** [Diaoul/subliminal#598]
Hi could not upgrade automatically:

```
pip install --upgrade subliminal
Downloading/unpacking subliminal from https://pypi.python.org/packages/source/s/subliminal/subliminal-1.1.1.tar.gz#md5=d9d4c3863d366d7dde498c6c318ae933
  Downloading subliminal-1.1.1.tar.gz
  Running setup.py egg_info for package subliminal
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/tmp/pip_build_root/subliminal/setup.py", line 11
        setup_requirements = ['pytest-runner'] if {'pytest', 'test', 'ptr'}.intersection(sys.argv) else []
                                                           ^
    SyntaxError: invalid syntax
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/tmp/pip_build_root/subliminal/setup.py", line 11

    setup_requirements = ['pytest-runner'] if {'pytest', 'test', 'ptr'}.intersection(sys.argv) else []

                                                       ^

SyntaxError: invalid syntax

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/subliminal
```

**Comment by Diaoul** [comment]
Are you using python 2.6? If so please update.

**Comment by tikismoke** [comment]
Crap you mean it's need 2.7 or 3?

**Comment by Diaoul** [comment]
Yes: https://github.com/Diaoul/subliminal/blob/master/setup.py#L52-L58

**Comment by tikismoke** [comment]
Crap didn't see it in RTFM :(

**Comment by Diaoul** [comment]
Fixed with 63dd6a1f3c4dd890b353c228ae9db1ab7ae75cb1

**Comment by Diaoul** [comment]
By fixed I mean the docs, not python 2.6 support ;)

**Comment by tikismoke** [comment]
Of course ;) will have to find time to migrate what still need 2.6 on this host. Thank's for all your work by the way.
